docs/DeveloperGuide.adoc

```diff
@@ -518,44 +518,44 @@ The List mechanism is facilitated by commands which use `Logic` interface to obt
+
 [source,java]
 ----
-public UniquePersonList getBlacklistedPersons() {
-    UniquePersonList blacklistedPersons = new UniquePersonList();
-    for (Person person : persons.getInternalList()) {
-        if (person.getIsBlacklisted()) {
-            try {
-                blacklistedPersons.add(person);
-            } catch (DuplicatePersonException e) {
-                assert false : "This is not possible as prior checks have been done";
-            }
-        }
-    }
-    return blacklistedPersons;
+public ObservableList<ReadOnlyPerson> asObservableBlacklist() {
+    return FXCollections.unmodifiableObservableList(mappedList.stream()
+            .filter(person -> person.isBlacklisted()).collect(toCollection(FXCollections::observableArrayList)));
 }
 ----
 
 Although this way of implementation seems inefficient, it supports robust synchronisation among the various other lists. For example, if a person is deleted from the masterlist, he will also be deleted from the blacklist. Likewise for various other commands that changes the `ReadOnlyPerson` object. Thus, it is efficient in this aspect. Moreover, this implementation sets the groundwork for future implementations of various other lists.
 
+One other important aspect to this implementation is that commands now have exclusive control over the list. This means that commands will only work on the persons in the current displayed list. For example, `find` command will only search and find persons in the current displayed list. It will not try to search in the masterlist, where all the contacts are stored. `sort` command will only sort the persons in the current displayed list. If the current displayed list is the blacklist, the output of the `sort` command will just be a list of sorted blacklisted persons.
+
+However, if a `Person` is modified by a command, the modification will be reflected across all other lists. For example, if a person exists in both the blacklist and the overdue list, modifications done by an `edit` command in the blacklist will also update the overdue list. These changes can be observed when the user switches to the overdue list using the `overduelist` command.
+
 ==== Design Considerations:
 
-**Aspect:** Other commands still work in Blacklist mode(`Add`, `Delete` command) +
-**Alternative 1 (current choice):** Inform user in user guide to use only specific commands for that list +
-**Pros:** Easy to implement. +
-**Cons:** Performance issues as users are bound to make a mistake. +
-**Alternative 2:** Restrict other command usages in certain lists +
-**Pros:** Users will never make a mistake sending the wrong person to the wrong list. +
-**Cons:** Implementation is complicated and thus implementation has to revisited whenever new commands are introduced.
+**Aspect:** Syncing of lists +
+**Alternative 1 (current choice):** Use tedious syncing methods to ensure lists are in sync +
```

Comment:
try not to use subjective words like tedious in the alternative itself. those words should be in pros and cons. 
maybe change to manual syncing methods? does that make sense?

Reply:
Yeapp noted.